doxygen/doxygen.bzl

```diff
@@ -36,18 +40,26 @@ def _swift_doxygen_impl(ctx):
 
         PRW=`pwd`
 
-        # backward compatibility with old CMake-style doxygen config files
-        sed -i "s|@DOXYGEN_DOT_FOUND@|$DOXYGEN_DOT_FOUND|g" {config}
-        sed -i "s|@DOXYGEN_DOT_PATH@|$DOXYGEN_DOT_PATH|g" {config}
-        sed -i "s|@PLANTUML_JAR_PATH@|/usr/local/bin/plantuml.jar|g" {config}
-        sed -i "s|@INPUT_DIR@|$PROJECT_SOURCE_DIR|g" {config}
-        sed -i "s|@PROJECT_NAME@|$PROJECT_NAME|g" {config}
-        sed -i "s|@STABLE_GIT_TAG@|$STABLE_GIT_TAG|g" {config}
-        sed -i "s|@DOXYGEN_EXCLUDE@|$DOXYGEN_EXCLUDE|g" {config}
-        sed -i "s|@PROJECT_SOURCE_DIR@|$PRW|g" {config}
+        # Apply backward compatibility sed replacements
+        sed "s|@DOXYGEN_DOT_FOUND@|$DOXYGEN_DOT_FOUND|g" {original_config} | \
```

Comment:
The chaining of `sed` commands changes the output file `processed_config` only once during this action`.